edx_management_commands/management_commands/urls.py

```diff
@@ -0,0 +1,15 @@
+"""
```

Comment:
Most external Django apps I see use a separate test project that has settings and urls.py. That way you don't need to add unnecessary files.